provider-gui/components/AccountPicker.tsx

```diff
@@ -0,0 +1,68 @@
+'use client'
+
+import { FormControl, FormGroup, MenuItem, Select } from '@mui/material'
+import { InjectedAccountWithMeta } from '@polkadot/extension-inject/types'
+import { SelectChangeEvent } from '@mui/material/Select'
+import { networks } from '@/types/GlobalStateTypes'
+import { useEffect, useState } from 'react'
+import { useGlobalState } from '../contexts/GlobalContext'
+import { web3Accounts, web3Enable } from '@polkadot/extension-dapp'
+import React from 'react'
+
+interface Injected {
+    enable: typeof web3Enable
+    accounts: typeof web3Accounts
+}
+
+const injected: Injected = {
+    enable: web3Enable,
+    accounts: web3Accounts,
+}
+
+const AccountPicker: React.FC = () => {
+    const [accounts, setAccounts] = useState<InjectedAccountWithMeta[]>([])
+    const { currentAccount, setCurrentAccount, network, setNetwork } = useGlobalState()
+
+    useEffect(() => {
+        const enableExtension = async () => {
+            await injected.enable('my cool dapp')
```

Comment:
"my cool dapp" should probs be changed :joy:

Reply:
Changed to Prosopo Provider GUI